File: apzkr-pzpi-21-5-vyshniak-viktoriia/Task1-Server/BLL/Infrastructure/Models/Notification/NotificationModel.cs
using BLL.Infrastructure.Models.Pet;

namespace BLL.Infrastructure.Models.Notification;
public class NotificationModel
{
    public Guid NotificationId { get; set; }

    public DateTime CreatedOnUtc { get; set; }

    public PetNotificationProfile PetProfile { get; set; }
}

File: apzkr-pzpi-21-5-vyshniak-viktoriia/Task4-MobileClient/app/src/main/java/com/pawpoint/model/response/NotificationModel.kt
package com.pawpoint.model.response

import com.google.gson.annotations.SerializedName

data class NotificationModel(

    @SerializedName("notificationId")
    val notificationId: String,

    @SerializedName("createdOnUtc")
    val createdOnUtc: String,

    @SerializedName("petProfile")
    val petProfile: PetNotificationProfile,
)
